src/rules/fileNameCasingRule.ts

```diff
@@ -43,12 +111,27 @@ export class Rule extends Lint.Rules.AbstractRule {
             * \`${Casing.SnakeCase}\`: File names must be snake-cased: \`file_name.ts\`.`,
         options: {
             type: "array",
-            items: [
-                {
-                    type: "string",
-                    enum: [Casing.CamelCase, Casing.PascalCase, Casing.KebabCase, Casing.SnakeCase],
-                },
-            ],
+            items: {
+                anyOf: [
+                    {
+                        type: "array",
+                        items: [
+                            {
+                                type: "string",
+                                enum: rules,
+                            },
+                        ],
+                    },
+                    {
+                        type: "object",
+                        // TODO: I am not sure how to setup the type
+                        // properties: {
+                        //     type: "string",
+                        //     enum: rules,
+                        // },
+                    },
+                ],
+            },
         },
         optionExamples: [
```

Comment:
Please update these and the plain-text description in the metadata to include your addition. It'd be a shame if folks didn't know about it!

Reply:
@JoshuaKGoldberg yeah sorry, I forgot to ask where do I need to update the documentation since I didn't understand the bullet point